packages/picasso-lab/src/Dropzone/story/Default.example.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Dropzone,
+  DropzoneErrorCode,
+  DropzoneFileUpload,
+  DropzoneFileError,
+  DropzoneFileRejection,
+  DropzoneDropEvent
+} from '@toptal/picasso-lab'
+
+type ValidatorType = (
+  file: File
+) => DropzoneFileError | DropzoneFileError[] | null
+type OnDropType = (
+  acceptedFiles: File[],
+  fileRejections: DropzoneFileRejection[],
+  event: DropzoneDropEvent
+) => void
+
+const MAX_SIZE = 600 * 1000
+
+const customSizeValidator: ValidatorType = file => {
```

Comment:
maybe make this to simplify? so we would always return array of `DropzoneFileError` even if single error has happened
```suggestion
const customSizeValidator: DropzoneFileError[] | null = (file: File) => {
```